Day 2/BMI Calculator.py

```diff
@@ -0,0 +1,4 @@
+height = input("enter your height in m: ")
+weight = input ("enter your weight in kg: ")
```

Comment:
`weight = input ("enter your weight in kg: ")` === > `weight = input("enter your weight in kg: ")`